Python Tutorial Exercises(Week 3)/#3.py

```diff
@@ -0,0 +1,22 @@
+'''
+This script takes an inputs number 's' and stores the number to the 'total_number' list 
+and adds a number to the 'week_number' list to count the weeks. When the 'total_number' 
+is greater than the goal then print the number of weeks that have passed and 
+the final amount he has saved once the goal has been reached.
+'''
+
+g = 1000
+week_number = 0
+total_number = 0
+s = input()
```

Comment:
This initial input from the user is never used since the `s = input("Enter a number: ")` in the while loop overrides whatever value was given